CorsixTH/Src/th_movie.cpp

```diff
@@ -311,7 +311,7 @@ void av_packet_queue::push(AVPacket* pPacket) {
   }
 #endif
 
-  AVPacketList* pNode = (AVPacketList*)av_malloc(sizeof(AVPacketList));
+  th_packet_list* pNode = (th_packet_list*)av_malloc(sizeof(th_packet_list));
```

Comment:
shouldn't this do `static_cast<th_packet_list *>(av_malloc(sizeof(th_packet_list));` ?

Reply:
Yeah, this whole class has a lot more cisms than it should.

Reply:
given your intentions, I am fine if you just merge this as-is